tools-local/tasks/mobile.tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -42,7 +43,7 @@ private class WasmAppConfig
         [JsonPropertyName("assembly_root")]
         public string AssemblyRoot { get; set; } = "managed";
         [JsonPropertyName("enable_debugging")]
-        public int EnableDebugging { get; set; } = 0;
```

Comment:
The previous logic was wrong here too but EnableDebugging is more like DebugLevel and is and integer.  The integer value should be passed to the call.

Reply:
Okay, I will rename EnableDebugging to DebugLevel and it's being passed to the call through mono-config.js.

Reply:
Sounds good.  DebugLevel > 0 means debugging is enabled and the value sets the log level.

Reply:
I just realized this going to break some things in other modules.  We should either also support the old name or move this part to a separate pr